### DISCUSSION TITLE
About Transforming Models

### DISCUSSION THREAD

**Discussion by UUTTR9** [RapidAI/RapidLatexOCR#4] [Q&A] [answered]
Hello, thank you very much for your work. I have previously trained the model on my own data, so I would like to ask for more details on how to convert the onnx model so that I can convert the model I trained into onnx

**Comment by SWHL** [comment]
- Is the model structure you trained consistent with the model structure of Latex-OCR?
- By the way, I need time to organize the detail of converting the Latex-OCR model.

**Comment by SWHL** [comment]
You can refer this [wiki](https://github.com/RapidAI/RapidLatexOCR/wiki/Model-Conversion-Notes) to convert your model.

**Comment by SWHL** [comment, marked as answer]
I have reorganized the conversion code into this project [ConvertLaTeXOCRToONNX](https://github.com/SWHL/ConvertLaTeXOCRToONNX).
Enjoy it!
